_includes/markdown/Environments.md

```diff
@@ -40,12 +40,41 @@ A special hostname `all` is available that will stop all running environments as
 
 A special hostname `all` is available that will start all environments as well as the global services.
 
+## Configure an Environment
+
+An environment's `config/` directory contains settings for various services:
+
+- `elasticsearch/`
+    - `elasticsearch.yml`
+- `nginx/`
+    - `default.conf`
+    - `develop.conf`
+    - `server.conf`
+- `php-fpm/`
+    - `docker-php-ext-xdebug.ini` — this is equivalent to `php.ini`, and is where you should put settings that would be found in a file named `php.ini`, such as `max_execution_time`, `memory_limit`, `xdebug.max_nesting_level`, `post_max_size`, `upload_max_filesize`, and similar.
+    - `wp-cli.develop.yml`
+    - `wp-cli.local.yml`
+
+After making changes to files in `config/` or to `docker-compose.yml`, you must recreate the containers. Instructions for doing so can be found on this page under [Restart an environment](#restart-an-environment)
+
 ## Restart an Environment
 
 `10updocker restart <hostname>` will restart all services associated with a preexisting environment.
 
 A special hostname `all` is available that will restart all environments as well as the global services.
 
+Restarting with `10updocker restart` does not apply configuration changes. If you've made changes to an environment's `docker-compose.yml` or the files in `/config` directory, you will need to stop the environment (but not global services) and then run [`docker-compose up`](https://docs.docker.com/compose/reference/up/) to recreate the containers:
```

Comment:
```suggestion
Restarting with `10updocker restart` does not apply configuration changes. If you've made changes to an environment's `docker-compose.yml` or the files in `/config` directory, you will need to stop the environment (but not global services) and then start it again to make containers re-read changes from the docker-compose file and re-mount configs:
```

Reply:
A simple `10updocker restart` does not work. The changes are not applied until `docker-compose up` is run.